rollup.config.js

```diff
@@ -0,0 +1,95 @@
+import babel from '@rollup/plugin-babel';
+import commonjs from '@rollup/plugin-commonjs';
+import nodeResolve from '@rollup/plugin-node-resolve';
+import replace from '@rollup/plugin-replace';
+import copy from 'rollup-plugin-copy';
+import esbuild from 'rollup-plugin-esbuild';
+import scss from 'rollup-plugin-scss';
+import * as sass from 'sass';
+
+const serverBundle = {
+	input: 'src/server/app.js',
+	output: {
+		file: 'built/main.js',
+		sourcemap: 'inline'
+	},
+	external: [
+		'express',
+		'express-session',
+		'node:http',
+		'prop-types',
+		'react',
+		'react-bootstrap-typeahead',
+		'react-dom/server',
+		'react-helmet',
+		'react-redux',
+		'react-router-dom',
+		'react-router-dom/server.js',
+		'redux',
+		'redux-thunk'
+	],
+	watch: {
+		clearScreen: false
+	},
+	plugins: [
+		esbuild({
+			jsxFactory: 'React.createElement',
+			jsxFragment: 'React.Fragment'
+		}),
```

Comment:
The properties for the object provided as an argument are taken from this usage example https://github.com/egoist/rollup-plugin-esbuild#usage, though for many properties the default value is appropriate.

```js
import esbuild from 'rollup-plugin-esbuild'

export default {
  plugins: [
    esbuild({
      // All options are optional
      include: /\.[jt]sx?$/, // default, inferred from `loaders` option
      exclude: /node_modules/, // default
      sourceMap: true, // default
      minify: process.env.NODE_ENV === 'production',
      target: 'es2017', // default, or 'es20XX', 'esnext'
      jsx: 'transform', // default, or 'preserve'
      jsxFactory: 'React.createElement',
      jsxFragment: 'React.Fragment',
      // Like @rollup/plugin-replace
      define: {
        __VERSION__: '"x.y.z"',
      },
      tsconfig: 'tsconfig.json', // default
      // Add extra loaders
      loaders: {
        // Add .json files support
        // require @rollup/plugin-commonjs
        '.json': 'json',
        // Enable JSX in .js files too
        '.js': 'jsx',
      },
    }),
  ],
}
```